tests/test_aaa_authorization_service.rb

```diff
@@ -632,240 +537,144 @@ def test_get_default_groups
     aaa_a_service.destroy
   end
 
-  def test_commands_default_set_groups
-    # Preconfigure tacacs, tacacs server and AAA valid group
-    group0 = 'tac_group'
-    preconfig_tacacs_server_access(group0)
-
-    # Preconfig for test
-    group1 = 'bxb100'
-    group2 = 'sjc200'
-    group3 = 'rtp10'
-    servers = [group1, group2, group3]
-    config_tacacs_servers(servers)
-
-    # Commands, service default
-    type_str = 'commands'
-    type = :commands
-    service = 'default'
-    aaa_a_service = AaaAuthorizationService.new(type, service)
-
-    # Single group, with method 'unselected'
-    method = :unselected
-    groups = [group0]
-    aaa_a_service.groups_method_set(groups, method)
+  def test_commands_default_unselected_single
+    aaa_a_service = AaaAuthorizationService.new(:commands, 'default')
+    aaa_a_service.groups_method_set(tacacs_groups[0], :unselected)
 
-    p = /#{prefix} #{type_str} #{service} group #{group0}/
+    p = get_pattern(:commands, 'default', tacacs_groups[0])
     assert_show_match(command: show_cmd, pattern: p)
+  end
 
-    # Multi group, with method 'unselected'
-    method = :unselected
-    groups = [group0, group1, group2]
-    aaa_a_service.groups_method_set(groups, method)
+  def test_commands_default_unselected_multi
+    aaa_a_service = AaaAuthorizationService.new(:commands, 'default')
+    aaa_a_service.groups_method_set(tacacs_groups[0..2], :unselected)
 
-    p = /#{prefix} #{type_str} #{service} group #{group0} #{group1} #{group2}/
+    p = get_pattern(:commands, 'default', tacacs_groups[0..2])
     assert_show_match(command: show_cmd, pattern: p)
+  end
 
-    # Multi group, with method 'local'
-    method = :local
-    groups = [group0, group1, group3]
-    aaa_a_service.groups_method_set(groups, method)
+  def test_commands_default_local_multi
+    aaa_a_service = AaaAuthorizationService.new(:commands, 'default')
+    aaa_a_service.groups_method_set(tacacs_groups[0..2], :local)
 
-    group_str = "group #{group0} #{group1} #{group3}"
-    p = /#{prefix} #{type_str} #{service} #{group_str} local/
+    p = get_pattern(:commands, 'default', tacacs_groups[0..2], :local)
```

Comment:
I changed it to use groups 0, 1, and 2 because `[0..2]` was faster than typing each individual group :-)